client/app/widgets/ToggleLocale/index.jsx

```diff
@@ -36,7 +37,12 @@ const onChange = (e: SyntheticEvent<HTMLInputElement>) => {
 export const ToggleLocale = (props: Props) => {
   const { locale, locales } = props;
   return (
-    <div aria-label={I18n.t('language')}>
+    <div>
+      <InputLabel
```

Comment:
Instead of using InputLabel here, let's create a prop in the Input component called hideLabel that's either undefined or a boolean. Then based on that in InputLabel which is called by displayLabel(), apply the visuallyHidden class to InputLabel if hideLabel is true. 

By applying this to the generic Input component, we can reuse this same logic for other Input components that have hidden labels.